### ISSUE TITLE
Create a sample/mock gallery app

### ISSUE THREAD

**Issue by gsathya** [gsathya/htpt#4]

**Comment by gsathya** [comment]
I think the best thing to do here is to create a static page with some images and serve it with Python's BaseHTTPServer.

**Comment by gsathya** [comment]
https://github.com/gsathya/htpt/tree/gallery Thoughts?

**Comment by gsathya** [comment]
```
$ git pull gsathya
$ git checkout gsathya/gallery
$ cd gallery
$ python -m SimpleHTTPServer
# go to localhost:8000
```

**Comment by gsathya** [comment]
can someone please review this?

**Comment by gsathya** [comment]
Hmm. I'm going to merge this.

**Comment by gsathya** [comment]
Fixed in 713d431e3b0c639f5471dc9d1c6c060d2295a874
